samples/snippets/ordering_mode_partial_test.py

```diff
@@ -16,30 +16,34 @@
 def test_quickstart() -> None:
     import bigframes.pandas
 
-    # We need a fresh session since we're modifying connection options.
-    bigframes.pandas.close_session()
+    try:
+        # We need a fresh session since we're modifying connection options.
```

Comment:
[nit] should we say "bigquery connection options" or "bigquery options" for clarity, wherever this code sample appears